src/features/theme-switcher/ui/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,19 @@
+import { Space, Switch } from "antd";
+import { useUnit } from "effector-react";
+import { $theme, themeChanged } from "../model";
+
+export const ThemeSwitcher = () => {
+  const [themeValue, themeChangedFn] = useUnit([$theme, themeChanged]);
+
+  const onChange = (checked: boolean) => {
+    checked ? themeChangedFn("dark") : themeChangedFn("light");
+  };
+
+  return (
+    <Space style={{color: '#fff'}}>
+      День
+      <Switch checked={themeValue === "dark"} onChange={onChange} style={{backgroundColor: '#d9d9d9'}}/>
```

Comment:
1. use css variables instead of '#fff'
2. use classmanes with css in js instead of style attr

Reply:
I think you should use Switch without color